content/develop/connect/insight/rdi-connector.md

```diff
@@ -0,0 +1,155 @@
+
+Each of these will be described in more detail in subsequent sections.
+
+There are also equivalent buttons at the top of the editor pane for the first two of these functions.
+
+<img src="../images/ri-rdi-pl-editor-minibuttons.png">
+
+If you'd rather work from a blank slate, exit the dialog, which will leave you in the **Configuration file** editor where you can begin editing your configuration pipeline.
+
+### Download pipelines from your RDI configuration
+
+Click the **Download from server** button in the **Start with your pipeline** dialog to download previously defined configuration and transformation job pipelines from your RDI server. The downloaded pipelines will be displayed in the **Pipeline management** pane. The configuration pipeline file will be displayed in center panel.
+
+<img src="../images/ri-rdi-pl-dl.png">
+
+### Upload pipelines from your local machine
+
+Click the **Upload from file** button in the **Start with your pipeline** dialog to upload your YAML files from your local machine. The files must be stored in a zip file that has the following structure.
+
+```
+├── config.yaml
+└── jobs
+    └── job1.yaml
+```
+
+The `config.yaml` file, your configuration pipeline YAML file, is required. The `jobs` directory can be empty, as job pipelines are not required, but the empty directory must exist in the zip file. Otherwise, the `jobs` folder might include one or more transformation job pipeline files.
+
+### Create a new configuration pipeline using the built-in editor
+
+Click the **Create new pipeline** button in the **Start with your pipeline** dialog to create new pipeline data using Redis Insight's built-in editor. After doing so, you'll enter the **Configuration file** editor and you'll see a **Select a template** dialog in the upper right-hand corner of the editor.
+
+<img src="../images/ri-rdi-pl-editor-dlg.png">
+
+Make your selections in the provided fields:
+
+- **Pipeline type** is set to **Ingest**. This is the only available option.
+- **Database type** has six options:
+  - mongodb
+  - cassandra
+  - mysql
+  - oracle
+  - postgresql
+  - sqlserver
+
+After you make your selections and click **Apply**, Redis Insight will populate the editor window with an appropriate template. To start from scratch, click **Cancel**.
+
+<img src="../images/ri-rdi-pl-editor-template.png">
+
+### Create a new transformation job pipeline using the built-in editor
+
+In the **Pipeline Management** pane, click the `+` next to the **Jobs** folder and enter a name for the new job.
+Next, click on the job name you just created.
+This will place you in the transformation job editor with the template selection menu open. There's only one template available: **Ingest**. Click **Apply** and Redis Insight will populate the editor windows with an appropriate template. To start from scratch, click **Cancel**.
```

Comment:
```suggestion
This will take you to the transformation job editor with the template selection menu open. There's only one template available: **Ingest**. Click **Apply** and Redis Insight will populate the editor windows with an appropriate template. To start from scratch, click **Cancel**.
```